src/components/QueryEditor/index.tsx

```diff
@@ -9,5 +9,9 @@ import { AdxDataSourceOptions, KustoQuery } from 'types';
 type Props = QueryEditorProps<AdxDataSource, KustoQuery, AdxDataSourceOptions>;
 
 export const QueryEditor: React.FC<Props> = (props) => {
-  return config.featureToggles.adxLegacyEditor ? <LegacyQueryEditor {...props} /> : <NewQueryEditor {...props} />;
+  return (config.featureToggles as any).adxLegacyEditor ? (
```

Comment:
I think this reads better as just a regular `if`/`else` that returns early if `true`.

Reply:
I've updated this, personally I'm more of an advocate for ternary operators over `if/else` as I find them simpler but maybe they're not as readable.